src/Tizen.NUI.Components/Controls/DropDown.cs

```diff
@@ -205,9 +205,118 @@ public enum ListOrientation
             Right,
         }
 
+        /// <summary>
+        /// Get or set header text.
+        /// </summary>
+        /// <since_tizen> 8 </since_tizen>
+        /// This will be public opened in tizen_6 after ACR done. Before ACR, need to be hidden as inhouse API.
+        public TextLabel HeaderText
```

Comment:
Hello,
is it OK not to add [EditorBrowsable .. Never] ? 
I can see other APIs have this. 
also please remove <since_tizen> tag and remove tizen_6 word. (please change as tizen_next.)
thank you.

Reply:
The class of DropDown has been hidden. So there is not ACR notice of public api added. I will add [Never] also.